Gems/WhiteBox/Code/Source/Viewport/WhiteBoxManipulatorViews.h

```diff
@@ -36,6 +36,7 @@ namespace WhiteBox
             AzFramework::DebugDisplayRequests& debugDisplay, const AzFramework::CameraState& cameraState,
             const AzToolsFramework::ViewportInteraction::MouseInteraction& mouseInteraction) override;
 
+        AZ::Transform m_transform;
```

Comment:
update: Can we give this a more descriptive name instead of just transform, as this could be confusing returning to this as it's really just an offset, could we call this `m_faceOffsetTransform` perhaps? (and add a comment to explain what it's used for). Thanks!

Reply:
or even better similar to the CVar name - `m_polygonViewOverlapOffset` (and just omit `transform` as it's the type, we don't always need to include the name of the type as we're statically typed anyway 🙂)